packages/react-vapor/src/components/popover/examples/PopoverConnectedExamples.tsx

```diff
@@ -1,8 +1,9 @@
 import * as React from 'react';
 
+import {ExampleComponent} from '../../../../docs/src/components/ComponentsInterface';
 import {PopoverConnected} from '../PopoverConnected';
 
-export const PopoverConnectedExamples = () => (
+const PopoverConnectedExamples = () => (
     <div className="mt2">
         <h1 className="text-blue mb1 bold">Popover Connected</h1>
         <PopoverConnected id="popover-connected-example1" attachment="top left" targetAttachment="bottom left">

```

Comment:
The example is quite weird, I see some things that could be done to improve it:
- Add the `inline-block` class to the button
- Remove the title `<h1 className="text-blue mb1 bold">Popover Connected</h1>` (this information is now in the header)
- Add a margin after the button so that the popover doesn't opend in a sneaky way (grey on grey) above the source code block. 
![image](https://user-images.githubusercontent.com/35579930/65890552-0bc85a00-e371-11e9-9bad-9b521b3344a1.png)